### PR TITLE
Refactor/grpc config

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

File: cmd/neofs-node/config.go [hunk 1]
```diff
@@ -348,10 +343,10 @@ func initCfg(path string) *cfg {
 		tlsKeyFile  string
 	)
 
-	if viperCfg.GetBool(cfgTLSEnabled) {
+	if grpcconfig.TLS(appCfg).Enabled() {
 		tlsEnabled = true
-		tlsCertFile = viperCfg.GetString(cfgTLSCertFile)
-		tlsKeyFile = viperCfg.GetString(cfgTLSKeyFile)
+		tlsCertFile = grpcconfig.TLS(appCfg).CertificateFile()
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config.go:348]: Lets calc `grpcconfig.TLS(appCfg)` once into var.

File: cmd/neofs-node/config/grpc/config.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+import "github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+
+const (
+	subsection    = "grpc"
+	tlsSubsection = "tls"
+
+	ListenAddressDefault = "127.0.0.1:50501"
+	TLSCertFileDefault   = ""
+	TLSKeyFileDefault    = ""
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/grpc/config.go:11]: Docs

File: cmd/neofs-node/config/grpc/config.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+)
+
+type TLSConfig struct {
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/grpc/config.go:14]: Docs

File: cmd/neofs-node/config/grpc/config.go [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	cfg *config.Config
+}
+
+// Endpoint returns value of "endpoint" config parameter
+// from "grpc" section.
+//
+// Returns ListenAddressDefault if value is not set.
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/grpc/config.go:21]: I think we must panic here.

File: cmd/neofs-node/config/grpc/config.go [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+// TLS returns structure that provides access to "tls" subsection of
+// "grpc" section.
+func TLS(c *config.Config) TLSConfig {
+	return TLSConfig{
+		cfg: c.Sub(subsection).Sub(tlsSubsection),
+	}
+}
+
+// Enabled returns value of "enabled" config parameter
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/grpc/config.go:39]: We don't need to specify parent sections here:
```suggestion
// Enabled returns value of "enabled" config parameter.
```

File: cmd/neofs-node/config/grpc/config.go [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+//
+// Returns false if value is not set.
+func (tls TLSConfig) Enabled() bool {
+	return config.BoolSafe(tls.cfg, "enabled")
+}
+
+// CertificateFile returns value of "certificate" config parameter
+// from "tls" subsection of "grpc" section.
+//
+// Returns TLSCertFileDefault if value is not set.
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/grpc/config.go:50]: I think we should panic then.

File: cmd/neofs-node/config/grpc/config.go [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+		return v
+	}
+
+	return TLSCertFileDefault
+}
+
+// KeyFile returns value of "key" config parameter
+// from "tls" subsection of "grpc" section.
+//
+// Returns TLSKeyFileDefault if value is not set.
```
Comment (cthulhu-rider) [inline cmd/neofs-node/config/grpc/config.go:63]: I think we should panic then.

File: cmd/neofs-node/config.go (hunk 1 above)
Comment (carpawell) [reply cmd/neofs-node/config.go]: moved in var section

File: cmd/neofs-node/config/grpc/config.go (hunk 2 above)
Comment (carpawell) [reply cmd/neofs-node/config/grpc/config.go]: these constants were deleted after fixing other conversations

File: cmd/neofs-node/config/grpc/config.go (hunk 3 above)
Comment (carpawell) [reply cmd/neofs-node/config/grpc/config.go:22]: added

File: cmd/neofs-node/config/grpc/config.go (hunk 5 above)
Comment (carpawell) [reply cmd/neofs-node/config/grpc/config.go]: fixed

File: cmd/neofs-node/config/grpc/config.go (hunk 6 above)
Comment (carpawell) [reply cmd/neofs-node/config/grpc/config.go]: added panic and test on it

File: cmd/neofs-node/config/grpc/config.go (hunk 7 above)
Comment (carpawell) [reply cmd/neofs-node/config/grpc/config.go]: added panic and test on it

**Review by alexvanin — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**